src/styles/app/15-all-docs.css

```
@@ -1,14 +1,36 @@
 .allDocsPane {
+	--all-docs-card-min-width: 184px;
+	--all-docs-card-min-height: 184px;
+	--all-docs-card-aspect: 1 / 1;
+	--all-docs-card-padding: 12px;
+	--all-docs-card-gap: 10px;
+	--all-docs-card-title-size: 0.9rem;
+	--all-docs-card-preview-size: 0.78rem;
+	--all-docs-card-meta-size: 0.68rem;
 	display: flex;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)